understanding/21/pointer-cancellation.html

```diff
@@ -14,6 +14,8 @@ <h3>Up-Event activation or completion</h3>
 			<p>The most accessible way to incorporate pointer cancellation is to make activation occur on the up-event.</p>
       
         <p>Up-event activation refers to the activation of a target when the pointer is released. In a touchscreen interaction, when the finger touches a target, the up-event activation only occurs when the finger is lifted while still being within the target boundary. Similarly in mouse interaction, the up-event occurs when the mouse button is released while the cursor is still within the boundary of the initial target set when the mouse button was pressed.</p>
+      <p>In contrast, the down-event can occur as soon as the pointer is pressed inside a target. Whether this actually happens can depend on the operating system, the user agent, and the mode of input. For example, the mousedown event will lead to a down-event in desktop operating systems when using mouse input, but will be ignored when using touch input in the same setting. Mousedown is currently ignored in mobile operating systems, but the down-event may be triggered here by the touchstart event, which (as the name suggests) is triggered by touch, but ignored when the same control is activated via mouse or trackpad input.</p>
```

Comment:
a few clarifications:

- it's not that `mousedown` is "ignored" on touch devices (nothing to do with "mobile operating systems" ... this isn't specific to the OS, but to the touchscreen technology). touch devices/browsers do fire compatibility mouse events (see https://w3c.github.io/touch-events/#mouse-events), which include `mousedown`; it's just that these are fired all in rapid sequence when the finger is lifted from the touchscreen (so in effect fired on the "up" event)